src/pages/index.js

```diff
@@ -21,23 +21,23 @@ export default function Home() {
 					siteName: NEXT_APP_NAME,
 				}}
 			/>
-			<div className="flex align-middle justify-center">
-				<div className="py-16 text-center">
+			<div className="flex flex-col sm:flex-row justify-center items-center">
+				<div className="py-8 sm:py-16 text-center">
```

Comment:
Need `sm:px-?` also to make padding in small screen

Reply:
Added `sm:px-8`